addons/account/wizard/account_secure_entries_wizard.py

```diff
@@ -0,0 +1,268 @@
+                if last_move_hashed:
+                    # remaining_moves either have a hash already or have a higher sequence_number than the last_move_hashed
+                    not_hashable_unlocked_moves = chain_info['remaining_moves'].filtered(
+                        lambda move: (not move.inalterable_hash
+                                      and move.sequence_number < last_move_hashed.sequence_number
+                                      and move.date > self.company_id.user_hard_lock_date)
+                    )
+                else:
+                    not_hashable_unlocked_moves = self.env['account.move']
+                chain_info['not_hashable_unlocked_moves'] = not_hashable_unlocked_moves
+                res.append(chain_info)
+        return res
+
+    @api.depends('company_id', 'company_id.user_hard_lock_date', 'hash_date')
+    def _compute_data(self):
+        for wizard in self:
+            unreconciled_bank_statement_line_ids = []
+            chains_to_hash = []
+            if wizard.hash_date:
+                for chain_info in wizard._get_chains_to_hash(wizard.company_id, wizard.hash_date):
+                    if 'unreconciled' in chain_info['warnings']:
+                        unreconciled_bank_statement_line_ids.extend(
+                            chain_info['moves'].statement_line_ids.filtered(lambda l: not l.is_reconciled).ids
+                        )
+                    else:
+                        chains_to_hash.append(chain_info)
+            wizard.unreconciled_bank_statement_line_ids = [Command.set(unreconciled_bank_statement_line_ids)]
+            wizard.chains_to_hash_with_gaps = [
+                {
+                    'first_move_id': chain['moves'][0].id,
+                    'last_move_id': chain['moves'][-1].id,
+                } for chain in chains_to_hash if 'gap' in chain['warnings']
+            ]
+
+            not_hashable_unlocked_moves = []
+            move_to_hash_ids = []
+            for chain in chains_to_hash:
+                not_hashable_unlocked_moves.extend(chain['not_hashable_unlocked_moves'].ids)
+                move_to_hash_ids.extend(chain['moves'].ids)
+            wizard.not_hashable_unlocked_move_ids = [Command.set(not_hashable_unlocked_moves)]
+            wizard.move_to_hash_ids = [Command.set(move_to_hash_ids)]
+
+    @api.depends('company_id', 'chains_to_hash_with_gaps', 'hash_date', 'not_hashable_unlocked_move_ids', 'max_hash_date', 'unreconciled_bank_statement_line_ids')
+    def _compute_warnings(self):
+        for wizard in self:
+            warnings = {}
+
+            if wizard.max_hash_date:
+                max_hash_date_string = format_date(self.env, wizard.max_hash_date)
+            else:
+                max_hash_date_string = _("today")
+            warnings['account_max_hash_date'] = {
+                'message': _("Posted entries are currently secured up to %s, inclusive.", max_hash_date_string),
```

Comment:
We don't want this message since it's the same date as the date set when opening the wizard (discussed with AVW/TSB)